.gitlab/ci/.gitlab-ci.global.yml

```
@@ -196,14 +198,28 @@
     - .default-cache
     - .default-before-script
 
-
 .trigger-slack-notification:
   stage: .post
   trigger:
     strategy: depend
     include:
       - local: .gitlab/ci/.gitlab-ci.slack-notify.yml
 
+.validate_content_test_conf_merged:
```

We care about a specific branch.
```suggestion
.validate_content_test_conf_branch_merged:
```